examples/pcaplay: integer divide by zero panic

When the execution time of the program is less than one second, a divide by zero exception is encountered in `examples/pcaplay/main.go` on line 93:

```go
fmt.Printf("Avg packet rate %d/s\n", packets/int(deltaTotal.Seconds()))
```

Panic output below.
```
panic: runtime error: integer divide by zero
[signal 0x8 code=0x1 addr=0x40320b pc=0x40320b]

goroutine 1 [running]:
panic(0x5f4460, 0xc8200100d0)
	/home/u/go/src/runtime/panic.go:481 +0x3e6
main.pcapInfo(0x7ffeb642fde6, 0x6, 0xecf75dd40, 0xffffffff131b4be0, 0x9cef20, 0xecf75dd40, 0x2598a628, 0x9cef20, 0x17, 0xc0f)
	/home/u/goget/src/github.com/google/gopacket/examples/pcaplay/main.go:93 +0x41b
main.main()
	/home/u/goget/src/github.com/google/gopacket/examples/pcaplay/main.go:127 +0x923
```

--- a/examples/pcaplay/main.go
+++ b/examples/pcaplay/main.go
@@ -90,7 +90,11 @@ func pcapInfo(filename string) (start time.Time, end time.Time, packets int, siz
 			}
 		}
 	}
-	fmt.Printf("Avg packet rate %d/s\n", packets/int(deltaTotal.Seconds()))
+	sec := int(deltaTotal.Seconds())
+	if sec == 0 {
+		sec = 1
+	}
+	fmt.Printf("Avg packet rate %d/s\n", packets/sec)
 	return start, end, packets, size
 }
 

